recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2.7.4.post1" %}
+{% set version = "2.8.0.post2" %}
 # TODO: Add archs 10.0 and 12.0 with CTK 12.8 release
```

Comment:
I would say let's wait the CUDA 12.9 rebuild for Pytorch at https://github.com/conda-forge/pytorch-cpu-feedstock/pull/393, and then we can try compiling with `TORCH_CUDA_ARCH_LIST=...;10.0;12.0+PTX`?